src/contact-summary/.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "es6": false,
+    "node": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 6
```

Comment:
Surely you mean 5? Or "es6" to be true? What am I missing here?

Reply:
I've been told that the code that goes into the app can't be es6 yet because not all projects are running the latest container. And the plan is to have a configuration flag that needs to be set to turn on es6 for a project.  https://github.com/medic/medic-conf/issues/109